GitUI/UserControls/FileStatusList.cs

```diff
@@ -1411,18 +1416,18 @@ private void FileStatusListView_DrawSubItem(object sender, DrawListViewSubItemEv
 
                 if (!string.IsNullOrEmpty(prefix))
                 {
-                    DrawString(textRect, prefix, SystemColors.GrayText);
+                    DrawString(textRect, prefix, item.Selected ? SystemColors.HighlightText : SystemColors.GrayText);
```

Comment:
This change loses gray / black contrast between directory and file name

Reply:
Merged.